bot/resources/evergreen/py_topics.yaml

```diff
@@ -67,23 +83,40 @@
 366674035876167691:
     - If you could wish for a library involving net-sec, what would it be?
 
-# software-testing
-463035728335732738:
+# software-design
+782713858615017503:
     -
 
 # tools-and-devops
 463035462760792066:
     - What editor would you recommend to a beginner? Why?
     - What editor would you recommend to be the most efficient? Why?
+    - How often do you use GitHub Actions and workflows to automate your repositories?
+    - What's your favorite app on GitHub?
+
+# unit-testing
+463035728335732738:
+    -
 
 # unix
 491523972836360192:
-    -
+    - What's your favorite Bash command?
+    - What's your most used Bash command?
+    - How often do you update your Unix machine?
+    - How often do you upgrade on production?
 
 # user-interfaces
 338993628049571840:
     - What's the most impressive Desktop Application you've made with Python so far?
+    - Have you ever made your own GUI? If so, how?
+    - Do you perfer Command Line Interfaces (CLI) or Graphic User Interfaces (GUI)?
+    - What's your favorite CLI (Command Line Interface) or TLI (Terminal Line Interface)?
```

Comment:
Is TLI a thing? Searched around a bit and only found a few uses, I believe TUI would be a more familiar and fitting term

Reply:
Sorry, misspelled. It's TUI, thanks for catching that!